conformity/fields/currency.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import, unicode_literals
+
+import attr
+import currint
+
+from conformity.error import (
+    Error,
+    ERROR_CODE_INVALID,
+)
+from conformity.fields.basic import Base
+
+
+@attr.s
+class Amount(Base):
+    """
+    currint.Amount instances
+    """
+
+    valid_currencies = attr.ib(default=currint.currencies.keys())
+    description = attr.ib(default=None)
```

Comment:
I thought we talked about having `gt`, `gte`, `lt`, `lte`? Seems reasonable, and it would make this field consistent with all of our other numeric types.

Reply:
I didn't want to give the impression that the comparators did any sort of currency conversion for you. Maybe we solve that with some documentation.